core/ui/compose/designsystem-biometric/src/main/kotlin/app/k9mail/core/ui/compose/designsystem/biometric/atom/textfield/TextFieldOutlinedPasswordBiometric.kt

```diff
@@ -0,0 +1,125 @@
+package app.k9mail.core.ui.compose.designsystem.biometric.atom.textfield
+
+import android.view.WindowManager
+import androidx.biometric.BiometricManager
+import androidx.biometric.BiometricPrompt
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.DisposableEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.fragment.app.FragmentActivity
+import app.k9mail.core.ui.compose.common.activity.LocalActivity
+import app.k9mail.core.ui.compose.designsystem.atom.textfield.TextFieldOutlinedPassword
+
+@Suppress("LongParameterList")
+@Composable
+fun TextFieldOutlinedPasswordBiometric(
```

Comment:
It should be documented that this has a hard dependency on the `FragmentActivity` similarly to `BiometricPasswordInput`

Reply:
Done. d839e4874571a1d18a513d10b2dc4cff67c99256